src/cmd/manage_nsfs.js

```diff
@@ -431,24 +541,36 @@ async function manage_account_operations(action, data, config_root, config_root_
 
 
 /**
- * list_config_file will list all the config files (json) in a given path
- * @param {fs.PathLike} full_path
+ * list_config_file will list all the config files (json) in a given config directory
+ * @param {string} config_path
  */
-async function list_config_file(full_path) {
-    const entries = await fs.promises.readdir(full_path);
+async function list_config_file(config_path) {
+    const entries = await fs.promises.readdir(config_path);
     const config_files = entries.filter(entree => entree.endsWith('.json'));
-    const resources = await P.map(config_files, config_file => get_config_data(full_path, config_file));
+    const resources = await P.map(config_files, config_file_name => {
+        const full_path = path.join(config_path, config_file_name);
+        return get_view_config_data(full_path);
+    });
+    return resources;
+}
+
+
+/**
+ * get_config_data will read a config file and return its content from it
+ * @param {fs.PathLike} config_file_path
+ */
+async function get_config_data(config_file_path) {
+    const data = await fs.promises.readFile(config_file_path);
+    const resources = JSON.parse(data.toString());
     return resources;
 }
 
 /**
- * get_config_data will get a json file and get the resource data from it
+ * get_view_config_data will read a config file and return its content ready to be printed
  * @param {fs.PathLike} config_file_path
- * @param {string} config_file_name
  */
-async function get_config_data(config_file_path, config_file_name) {
-    const full_path = get_config_file_path(config_file_path.toString(), config_file_name);
-    const data = await fs.promises.readFile(full_path);
+async function get_view_config_data(config_file_path) {
+    const data = await fs.promises.readFile(config_file_path);
     const resources = _.omit(JSON.parse(data.toString()), ['access_keys']);
```

Comment:
We can call `get_config_data` here:

```suggestion
    const resources = _.omit(get_config_data(config_file_path), ['access_keys']);
```

Reply:
yes I wanted to do that but then I needed to stringify data again..